docs/sources/rules/_index.md

```diff
@@ -117,25 +68,99 @@ groups:
+
+At the time of writing, these are the compatible backends that support this:
+
+- [Prometheus](https://prometheus.io/docs/prometheus/latest/disabled_features/#remote-write-receiver) (`>=v2.25.0`):
+  Prometheus is generally a pull-based system, but since `v2.25.0` has allowed for metrics to be written directly to it as well.
+- [Cortex](https://cortexmetrics.io/docs/api/#remote-write)
+- [Thanos (`Receiver`)](https://thanos.io/tip/components/receive.md/)
+
+Here is an example remote-write configuration for sending to a local Prometheus instance:
 
-Many nascent projects, apps, or even companies may not have a metrics backend yet. We tend to add logging support before metric support, so if you're in this stage, alerting based on logs can help bridge the gap. It's easy to start building Loki alerts for things like _the percentage of error logs_ such as the example from earlier:
 ```yaml
-- alert: HighPercentageError
-  expr: |
-  sum(rate({app="foo", env="production"} |= "error" [5m])) by (job)
-    /
-  sum(rate({app="foo", env="production"}[5m])) by (job)
-    > 0.05
+ruler:
+  ... other settings ...
+  
+  remote_write:
+    enabled: true
+    client:
+      url: http://localhost:9090/api/v1/write
 ```
 
+Further configuration options can be found [here](/configuration#ruler_config).
+
+### Resilience & Durability
+
+Given the above remote-write configuration, one needs to take into account what would happen if the remote-write receiver
+becomes unavailable.
+
+The Ruler component ensures some durability guarantees by buffering all outgoing writes in an in-memory queue. This queue
+holds all metric samples that are due to be written to the remote-write receiver, and while that receiver is down the buffer
+will grow in size.
+
+Once the queue is full, the oldest samples will be evicted from the queue. The size of this queue is controllable globally,
+or on a per-tenant basis, with the [`ruler_remote_write_queue_capacity`](/configuration#limits_config) limit setting. By default, this value is set to 10000 samples.
+
+**NOTE**: this queue only exists in-memory at this time; there is no Write-Ahead Log (WAL) functionality available yet.
+This means that if your Ruler instance crashes, all pending metric samples in the queue that have not yet been written will be lost.
+
+### Operational Considerations
+
+A number of metrics are available to monitor recording rule evaluations and writes.
+
+| Metric  | Description  |
+|---|---|
+| `recording_rules_samples_queued_current`  | Number of samples queued to be remote-written.                                 |
+| `recording_rules_samples_queued_total`    | Number of samples queued in total.                                             |
+| `recording_rules_samples_queue_capacity`  | Number of samples that can be queued before eviction of oldest samples occurs. |
+| `recording_rules_samples_evicted_total`   | Number of samples evicted from queue; queue is full!                           | 
```

Comment:
```suggestion
| `recording_rules_samples_queued_total`    | Total number of samples queued.                                             |
| `recording_rules_samples_queue_capacity`  | Number of samples that can be queued before eviction of the oldest samples occurs. |
| `recording_rules_samples_evicted_total`   | Number of samples evicted from queue because the queue is full.                           | 
```